apollo-router/tests/snapshots/tracing_tests__traced_basic_composition.snap

```diff
@@ -246,31 +246,7 @@ expression: get_spans()
                       }
                     }
                   },
-                  "children": {
-                    "apollo_router::query_planner::bridge_query_planner::parse_query": {
-                      "name": "apollo_router::query_planner::bridge_query_planner::parse_query",
-                      "record": {
-                        "entries": [
-                          [
-                            "otel.kind",
-                            "INTERNAL"
-                          ]
-                        ],
-                        "metadata": {
-                          "name": "parse_query",
-                          "target": "apollo_router::query_planner::bridge_query_planner",
-                          "level": "INFO",
-                          "module_path": "apollo_router::query_planner::bridge_query_planner",
-                          "fields": {
-                            "names": [
-                              "otel.kind"
-                            ]
-                          }
-                        }
-                      },
-                      "children": {}
-                    }
-                  }
+                  "children": {}
```

Comment:
is it expected ?

Reply:
the `parse_query` span was removed, I added it back in 600fce68b but now it's under the query analysis module